gmplot/gmplot.py

```diff
@@ -429,9 +440,12 @@ def write_polygon(self, f, path, settings):
     def write_heatmap(self, f):
         for heatmap_points, settings_string in self.heatmap_points:
             f.write('var heatmap_points = [\n')
-            for heatmap_lat, heatmap_lng in heatmap_points:
-                f.write('new google.maps.LatLng(%f, %f),\n' %
-                        (heatmap_lat, heatmap_lng))
+            for lat, lng, weight in heatmap_points:
+                location = 'new google.maps.LatLng(%f, %f)' % (lat, lng)
+                if weight == self.HEATMAP_DEFAULT_WEIGHT:
```

Comment:
should we just write out the weight regardless, even if it's just the default?

Reply:
I don't think so; it seems like that'll just inflate the size of the HTML file for no gain.